### ISSUE TITLE
Unable to unset environment variables

### ISSUE THREAD

**Issue by mbentley** [docker/machine#2287]
I just noticed that I am not able to unset my environment variables:

```
$ docker-machine env -u
Error trying to get host "": Loading host from store failed: open /Users/mbentley/.docker/machine/machines/config.json: no such file or directory
```

For whatever reason, it seem to be interpreting my command as `docker-machine env ""`:

```
$ docker-machine env ""
Error trying to get host "": Loading host from store failed: open /Users/mbentley/.docker/machine/machines/config.json: no such file or directory
$ docker-machine env -u ""
Error trying to get host "": Loading host from store failed: open /Users/mbentley/.docker/machine/machines/config.json: no such file or directory
```

System Info:
OS X 10.11.1
`docker-machine version 0.5.0 (04cfa58)`
VirtualBox: 5.0.10r104061

**Comment by jeanlaurent** [comment]
This is a known issue, the fix is currently scheduled for 0.5.1 which should be out sometimes next week.

Closing as duplicate of #2204

**Comment by mbentley** [comment]
Ah thanks; I should have checked closed issues as well.
